db/sql_stmt_cache.c

```diff
@@ -581,15 +584,6 @@ int stmt_cache_put_int(struct sqlthdstate *thd, struct sqlclntstate *clnt,
         }
     }
 
-    if ((rec->status & CACHE_HAS_HINT) && (rec->status & CACHE_FOUND_STR)) {
-        char *k = rec->cache_hint;
-        Pthread_mutex_lock(&gbl_sql_lock);
-        {
-            lrucache_release(sql_hints, &k);
```

Comment:
Why? Do we release it somewhere else?

Reply:
Yes we release [here](https://github.com/bloomberg/comdb2/blob/master/db/sql_stmt_cache.c#L631).

Reply:
okay